publish-hook/src/main/java/com/redhat/lightblue/hook/publish/PublishHook.java

```
@@ -133,25 +127,29 @@ public void processHook(EntityMetadata entityMetadata, HookConfiguration hookCon
         return rootIdentities;
     }
 
-    private void insert(String entityName, Object entity) throws ClassNotFoundException, IllegalAccessException, InvocationTargetException, IOException,
-            NoSuchMethodException, InstantiationException {
+    private void insert(String entityName, Object entity) {
         ObjectNode jsonNode = new ObjectNode(JsonNodeFactory.instance);
         jsonNode.put("entity", entityName);
         ArrayNode data = jsonNode.putArray("data");
         data.add(new ObjectMapper().valueToTree(entity));
 
         InsertionRequest ireq = InsertionRequest.fromJson(jsonNode);
-        Response r = lightblueFactory.getMediator().insert(ireq);
+        Response r;
+        try {
+            r = lightblueFactory.getMediator().insert(ireq);
+        } catch (ClassNotFoundException | IllegalAccessException | InvocationTargetException | NoSuchMethodException | InstantiationException | IOException e) {
+            throw Error.get(HOOK_NAME + ":MediatorException", e);
```

This means there is a configuration error. You can throw a runtime exception for this one.